src/Microsoft.AspNetCore.Blazor.Build/Microsoft.AspNetCore.Blazor.Build.nuspec

```diff
@@ -11,10 +11,10 @@
     </dependencies>
   </metadata>
   <files>
-    <file src="_._" target="lib/netstandard1.0/" />
+    <file src="$publishdir$netcoreapp2.1\Microsoft.AspNetCore.Blazor.TagHelperWorkaround.dll" target="lib/netstandard1.0/" />
```

Comment:
This will be referenced as part of Blazor.Build. Since we do private assets there, it won't be transitive.

Reply:
When we no longer need this workaround we can just remove the assembly from here. Since there's no code a user can see in this assembly it won't be needed.

Reply:
OK good